ui-tests/src/test/java/org/openmrs/reference/page/MarkPatientDeceasedPage.java

```diff
@@ -0,0 +1,114 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class MarkPatientDeceasedPage extends Page {
+	
+	private static final String URL = "/coreapps/markPatientDead.page";
+	private static final By DICTIONARY = By.id("dictionaryNavLink");
+	private static final By ADD_NEW_CONCEPT = By.linkText("Add new Concept");
+	private static final By CONCEPT_NAME = By.id("namesByLocale[en].name");
+	private static final By ADD_ANSWER= By.cssSelector("#codedDatatypeRow > td > table > tbody > tr > td.buttons > input:nth-child(1)");
+	private static final By CAUSEBOX = By.id("newAnswerConcept");
+	private static final By FEVER = By.xpath("/html/body/ul[1]/li[2]/a");
+	private static final By ADD_BUTTON = By.xpath("/html/body/div[5]/div[11]/button[1]");
+	private static final By SAVE = By.cssSelector("#saveDeleteButtons > input:nth-child(1)");
+	private static final By SEARCH = By.id("inputNode");
+	private static final By COD = By.xpath("//*[@id=\"openmrsSearchTable\"]/tbody/tr[1]/td/span");
+	private static final By NEW_COD_ID = By.xpath("//*[@id=\"conceptTable\"]/tbody/tr[1]/td");
+	private static final By ADMINISTRATION = By.id("administrationNavLink");
```

Comment:
Line 22 .is this administrationPage or administration link.

Reply:
`<li id="administrationNavLink">
			<a href="/openmrs/admin/index.htm">Administration</a>
		</li>`

Reply:
Do you mind sharing a  url link to this where you are inspecting from please, want to confirm .Thanks

Reply:
https://qa-refapp.openmrs.org/openmrs/dictionary/index.htm   right click on administration and inspect element

Reply:
> https://qa-refapp.openmrs.org/openmrs/dictionary/index.htm right click on administration and inspect element

Thanks , since administration link you have provided returns the same thing with administrationPage from this link https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java  , why not simply using this method